game-engine/assets/example-state/console.txt

```diff
@@ -27,10 +27,10 @@ Player 2 :H=300 S=100
 ██░░░░░░▓▓▓▓░░▓▓▓▓▓▓▓▓▓▓░░▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░▓▓▓▓▓▓▓▓▓▓░░▓▓▓▓░░░░░░██
 ██░░░░░░░░▓▓░░▓▓▓▓░░░░▓▓▓▓▓▓▓▓░░▓▓░░▓▓▓▓▓▓▓▓░░░░▓▓▓▓░░▓▓░░░░░░░░██
 ████░░░░░░░░░░░░░░░░░░░░░░▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░░░░░░░░░░░░░░░░████
-██████▓▓░░░░░░░░░░░░░░▓▓░░░░▓▓▓▓▓▓▓▓▓▓░░░░▓▓░░░░░░░░░░░░░░▓▓██████
-████████░░░░▓▓▓▓▓▓▓▓▓▓▓▓░░░░▓▓▓▓░░▓▓▓▓░░░░▓▓▓▓▓▓▓▓▓▓▓▓░░░░████████
-████████░░▓▓▓▓▓▓░░░░░░▓▓▓▓▓▓▓▓░░▓▓░░▓▓▓▓▓▓▓▓▓▓░░░░░░▓▓▓▓░░████████
-████████████▓▓▓▓░░22░░▓▓░░▓▓▓▓░░▓▓░░▓▓▓▓░░░░▓▓░░11░░▓▓████████████
-██████████████▓▓░░░░░░▓▓░░▓▓▓▓░░░░░░▓▓▓▓░░░░▓▓░░░░░░██████████████
-████████████████▓▓▓▓▓▓▓▓▓▓▓▓░░▓▓░░▓▓░░▓▓▓▓░░▓▓▓▓▓▓████████████████
+██████▓▓░░░░▓▓▓▓▓▓▓▓▓▓▓▓░░░░▓▓▓▓▓▓▓▓▓▓░░░░▓▓▓▓▓▓▓▓▓▓▓▓░░░░▓▓██████
+████████░░░░▓▓░░░░░░▓▓▓▓░░░░▓▓▓▓░░▓▓▓▓░░░░▓▓▓▓░░░░░░▓▓░░░░████████
+████████░░▓▓▓▓░░22░░▓▓▓▓▓▓▓▓▓▓░░▓▓░░▓▓▓▓▓▓▓▓▓▓░░11░░▓▓▓▓░░████████
+████████████▓▓░░░░░░▓▓░░░░▓▓▓▓░░▓▓░░▓▓▓▓░░░░▓▓░░░░░░▓▓████████████
+██████████████▓▓▓▓▓▓▓▓░░░░▓▓▓▓░░░░░░▓▓▓▓░░░░▓▓▓▓▓▓▓▓██████████████
+████████████████░░▓▓░░░░▓▓▓▓░░▓▓░░▓▓░░▓▓▓▓░░░░▓▓░░████████████████
 ██████████████████████▓▓▓▓░░░░▓▓▓▓▓▓░░░░▓▓▓▓██████████████████████
```

Comment:
This shows that even other worms had incorrect positions, but they were all offset in a similar way so it wasn't immediately obvious